data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+			`	"Here is where the Sheragi came to an end. The tools that drove them to extinction are long gone. We witnessed as they cast those tools into the heart of this star."`
+			choice
+				`	"You mean this ship? Its weapons systems seem to be intact."`
+					goto cannon
+				`	"What tools?"`
+			`	"Ones capable of crippling an entire world's worth of life."`
+				goto next
+			label cannon
+			`	"That ship is just a relic. The weapon that perished the Sheragi was indiscriminate. An irrational decision they made themselves."`
+			label next
+			`	As that is said, you're overcome by glimpses of memories that are not yours, not clear enough for you to identify what exactly is happening, but you can feel the sadness in them.`
+			`	The voice continues: "I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you wish to possess it, then take it. Perhaps the sight of that vessel will remind the living races of those who have come and gone... and how, despite their great accomplishments, their decision to make war reduced them to graveyards, relics, and memories."`
+			`	The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it.`
+
+	npc board
+		government "Sheragi"
+		personality derelict mute
+		system Zubenelhakrabi
+		ship "Emerald Sword" "Emerald Sword"
+		conversation
+			`You dock with the Emerald Sword, and the crew starts working to carefully find a way in, directed by the Hai archaeologists. There is no atmosphere inside it as the life support has long since ceased to work, and it takes a long time to restore it through your ship's own life support and patch tiny micrometeorite holes. Once it's in breathable conditions, it still smells like a tomb. You find many Sheragi skeletons and mummified remains. A few of them have prosthetics that survived better than their host. Your crew gets to work cleaning up the thick layer of dust and grime that covers every surface, while the more specialized crew tries to bring the power core back to life and take advantage of the zero-G environment to move the jump drive into the ship more easily. You'll need to land to properly install it, however. All that is left for you to do is relocate the many corpses littering the living spaces with your crew.`
+	
+	on complete
+		conversation
+			`It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere but flat enough to land on. You eventually park your ships over a mesa. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be otherwise unbreathable for long periods of time.`
```

Comment:
The "chlorine gas in the atmosphere" plot point seems to just be tossed into this conversation. Going off of my previous comment of making this feel more impactful, I'd perhaps change this to something along the lines of,
```suggestion
			`As you enter the atmosphere of the alien world, you begin running some scans of the air quality to determine if it is safe to breathe. You quickly learn why it is that the Sheragi perished in space; the atmosphere has a high concentration of chlorine gas, fatal to all living creatures aside from those that seem to have miraculously evolved here. It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere, but flat enough to land on. You eventually park your ships over a mesa. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be otherwise unbreathable for long periods of time.`
```